system/nagios-status-check/README.md

```diff
@@ -0,0 +1,69 @@
+# SYSTEM Nagios status check SignalFx detectors
+
+## How to use this module
+
+```hcl
+module "signalfx-detectors-nagios-status-check" {
+  source      = "github.com/claranet/terraform-signalfx-detectors.git//system/nagios-status-check?ref={revision}"
+
+  environment = var.environment
+  notifications = var.notifications
+}
+
+```
+
+## Purpose
+
+Creates SignalFx detectors with the following checks:
+
+- Custom script status
+
+This module has been designed to alert with the same behavior as [Nagios](https://nagios-plugins.org/doc/guidelines.html#AEN78), basically a gauge which is equal to 1 will trigger à `WARNING` and equal to 2 a `CRITICAL`. 
+
+On the agent side, you need to send your metrics with a metric named `gauge.status` in SignalFx, the `plugin_instance` dimension will allow you to find which script is failing. The easiest way to do that is to use the [exec collectd plugin](https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-custom.html).
+
+Here is an example : 
+
+```yaml
+- type: collectd/custom
+  intervalSeconds: 30
+  template: |
+    LoadPlugin exec
+    <Plugin exec>
+       Exec "signalfx-agent" "/usr/local/bin/check-generic-status-nagios-plugin.sh" "/usr/local/bin/scripts/custom-nagios-script.pl"
+    </Plugin>
+```
+
+The following script (`check-generic-status-nagios-plugin.sh` in the example above) takes a script path as an argument, execute it, check the output and send the status in the collectd format.
+
+```bash
```

Comment:
why not put this script as file in this directory and forward to it from this readme ?

Reply:
Not needed any more with `telegraf/exec`.